MainHomework/MainHomework/AllCharactersViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ViewController.swift
+//  MainHomework
+//
+//  Created by Наталья Владимировна Пофтальная on 11.11.2023.
+//
+
+import UIKit
+
+final class ViewController: UIViewController {
+    
+   private var collectionView: UICollectionView!
+    
+   private var layout: UICollectionViewFlowLayout!
+    
+   private var characters: [Hero] = []
+    
+   private let networkLoader = NetworkService()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupCollectionView()
+        loader()
+    }
+    
+   private func loader() {
```

Comment:
В названии методов лучше использовать глаголы 
```suggestion
   private func loadCharacters() {
```